UPGRADE.md

```diff
@@ -1,5 +1,13 @@
 # Upgrade to 3.0
 
+## BC BREAK: Changes in the `Doctrine\DBAL\Schema` API
+
+- Method `Doctrine\DBAL\Schema\AbstractSchemaManager::_getPortableViewDefinition()` no longer optionally returns false. It will always return a `Doctrine\DBAL\Schema\View` instance.
```

Comment:
Nitpicking: although it's a change, it's not a breaking change.

Reply:
Also, should the removed methods from the description be listed here?

Reply:
Doesn't hurt. I will add.